code/game/jobs/job/medical.dm

```diff
@@ -98,14 +98,8 @@
 		H.equip_to_slot_or_del(new /obj/item/clothing/mask/surgical(H), slot_wear_mask)
 		H.equip_to_slot_or_del(new /obj/item/device/pda/viro(H), slot_belt)
 	else if(has_alt_title(H, alt_title,"Nurse"))
-		if(H.gender == FEMALE)
-			if(prob(50))
-				H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nursesuit(H), slot_w_uniform)
-			else
-				H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nurse(H), slot_w_uniform)
-			H.equip_to_slot_or_del(new /obj/item/clothing/head/nursehat(H), slot_head)
-		else
-			H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/medical/purple(H), slot_w_uniform)
+		H.equip_to_slot_or_del(new /obj/item/clothing/head/nursehat(H), slot_head)

```

Comment:
If you dont have the gender check from before, every nurse (even the male ones) get the nurse hat.
Is that suitable for them aswell ?